modules/access_keys/access_keys.js

```diff
@@ -0,0 +1,30 @@
+exports.collections = function () {
+  const access_keys = {
+    _id: 'access_keys',
+    schema: {
+      type: 'object',
+      additionalProperties: false,
+      properties: {
+        user_id: {
```

Comment:
with the more recent changes you will need to add property `user_collection` to cater for users that are not necessarilly `users`